docker/Dockerfile.testing

```
@@ -11,9 +11,11 @@ USER root
 # Use 'update' for existing and 'install' for new package.
 # Keep same channel ordering to not revert anything.
 RUN umask 0000 \
-    && mamba install -c conda-forge -c cdat -c bokeh -c plotly -c pyviz/label/dev -c defaults -n birdy xscen==0.8.2 ravenpy==0.13.0 pydantic==2.6.3 \
-    && mamba clean --all --yes \
-    && pip install --no-cache-dir --upgrade figanos
+    && pip uninstall -y ravenpy \
+    && pip install --no-cache-dir git+https://github.com/CSHS-CWRA/RavenPy.git@1977732c8dbf39ea7d563e7e30052707ba8fb2ec \
```

I think you can use the git tag for simplicity:
```suggestion
    && pip install --no-cache-dir git+https://github.com/CSHS-CWRA/RavenPy.git@v0.13.1 \
```

---

No, using the real release on PyPI now, see a44255e22b5df7dc089436f3c1917934eb0374b8.  That commit hash was for testing only.